src/simulation/simulation_state.jl

```diff
@@ -52,23 +28,28 @@ function set_current_time!(s::SimulationState, val::Dates.DateTime)
 end
 
 get_decision_states(s::SimulationState) = s.decision_states
-get_system_state(s::SimulationState) = s.system_state
+get_system_states(s::SimulationState) = s.system_state
```

Comment:
Should `SimulationState.system_state` be changed to `SimulationState.system_states`?